MapboxNavigation/FeedbackViewController.swift

```diff
@@ -80,14 +113,18 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
         view.backgroundColor = .white
         progressBar.barColor = #colorLiteral(red: 0.9347146749, green: 0.5047877431, blue: 0.1419634521, alpha: 1)
         enableDraggableDismiss()
+        
+        let uuid = eventsManager.recordFeedback()
+        sendFeedbackHandler = defaultSendFeedbackHandler(uuid: uuid)
+        dismissFeedbackHandler = defaultDismissFeedbackHandler(uuid: uuid)
```

Comment:
Instead of currying closures within `defaultSendFeedbackHandler(uuid:)` and `defaultDismissFeedbackHandler(uuid:)`, pull those methods into simple `send(_:withUUID:)` and `dismiss(_:)` methods and store `uuid` on the class instead. Then, instead of calling `sendFeedbackHandler`, call `send(_:withUUID:)`.